keras/applications/regnet.py

```diff
@@ -0,0 +1,1549 @@
+      input_shape=input_shape,
+      pooling=pooling,
+      classes=classes,
+      classifier_activation=classifier_activation
+  )
+
+
+@keras_export("keras.applications.regnet.RegNetY120",
+              "keras.applications.RegNetY120")
+def RegNetY120(model_name="regnety120",
+               include_top=True,
+               include_preprocessing=True,
+               weights=None,
+               input_tensor=None,
+               input_shape=None,
+               pooling=None,
+               classes=1000,
+               classifier_activation='softmax'):
+  return RegNet(
+      MODEL_CONFIGS["y120"]["depths"],
+      MODEL_CONFIGS["y120"]["widths"],
+      MODEL_CONFIGS["y120"]["group_width"],
+      "Y",
+      MODEL_CONFIGS["y120"]["default_size"],
+      model_name=model_name,
+      include_top=include_top,
+      include_preprocessing=include_preprocessing,
+      weights=weights,
+      input_tensor=input_tensor,
+      input_shape=input_shape,
+      pooling=pooling,
+      classes=classes,
+      classifier_activation=classifier_activation
+  )
+
+
+@keras_export("keras.applications.regnet.RegNetY160",
+              "keras.applications.RegNetY160")
+def RegNetY160(model_name="regnety160",
+               include_top=True,
+               include_preprocessing=True,
+               weights=None,
+               input_tensor=None,
+               input_shape=None,
+               pooling=None,
+               classes=1000,
+               classifier_activation='softmax'):
+  return RegNet(
+      MODEL_CONFIGS["y160"]["depths"],
+      MODEL_CONFIGS["y160"]["widths"],
+      MODEL_CONFIGS["y160"]["group_width"],
+      "Y",
+      MODEL_CONFIGS["y160"]["default_size"],
+      model_name=model_name,
+      include_top=include_top,
+      include_preprocessing=include_preprocessing,
+      weights=weights,
+      input_tensor=input_tensor,
+      input_shape=input_shape,
+      pooling=pooling,
+      classes=classes,
+      classifier_activation=classifier_activation
+  )
+
+
+@keras_export("keras.applications.regnet.RegNetY320",
+              "keras.applications.RegNetY320")
+def RegNetY320(model_name="regnety320",
+               include_top=True,
+               include_preprocessing=True,
+               weights=None,
+               input_tensor=None,
+               input_shape=None,
+               pooling=None,
+               classes=1000,
+               classifier_activation='softmax'):
+  return RegNet(
+      MODEL_CONFIGS["y320"]["depths"],
+      MODEL_CONFIGS["y320"]["widths"],
+      MODEL_CONFIGS["y320"]["group_width"],
+      "Y",
+      MODEL_CONFIGS["y320"]["default_size"],
+      model_name=model_name,
+      include_top=include_top,
+      include_preprocessing=include_preprocessing,
+      weights=weights,
+      input_tensor=input_tensor,
+      input_shape=input_shape,
+      pooling=pooling,
+      classes=classes,
+      classifier_activation=classifier_activation
+  )
```

Comment:
To follow out other applications, include `preprocess_input` and `decode_predictions` symbols. `preprocess_input` should just be a passthrough function as you are using rescaling in the model.